package.json

```
@@ -12,6 +12,7 @@
   "devDependencies": {
     "eslint": "7.31.0",
     "eslint-config-htmlacademy": "4.0.2",
+    "nanoid": "^3.1.22",
```

версии библиотек фиксируем без модификаторов
```suggestion
    "nanoid": "3.1.22",
```